episodes/vaccine-comparisons.Rmd

```diff
@@ -0,0 +1,968 @@
+
+The code below details how the interventions are simulated:  
+```{r}
+early_start <- 0
+late_start <- 50
+duration <- 100
+time_end <- 200
+
+# close schools early
+close_schools_early <- intervention(
+  name = "School closure",
+  type = "contacts",
+  time_begin = early_start,
+  time_end = early_start + duration,
+  reduction = matrix(c(0.5, 0.01, 0.01))
+)
+
+# vaccination late
+vacc_late <- vaccination(
+  name = "vaccinate late",
+  time_begin = matrix(late_start, nrow(contact_matrix)),
+  time_end = matrix(late_start + duration, nrow(contact_matrix)),
+  nu = matrix(c(0.01, 0.01, 0.01))
+)
+
+# npis started early + vaccination late
+output_npi_early_vacc_late <- model_default(
+  population = uk_population,
+  transmission_rate = transmission_rate,
+  infectiousness_rate = infectiousness_rate,
+  recovery_rate = recovery_rate,
+  vaccination = vacc_late,
+  intervention = list(contacts = close_schools_early),
+  time_end = time_end, increment = 1.0
+)
+```
+
+
+
+```{r, echo = FALSE}
+# vaccinate early
+vacc_early <- vaccination(
+  name = "vaccinate early",
+  time_begin = matrix(early_start, nrow(contact_matrix)),
+  time_end = matrix(early_start + duration, nrow(contact_matrix)),
+  nu = matrix(c(0.01, 0.01, 0.01))
+)
+
+# npis only (early)
+output_npi_early <- model_default(
+  population = uk_population,
+  transmission_rate = transmission_rate,
+  infectiousness_rate = infectiousness_rate,
+  recovery_rate = recovery_rate,
+  intervention = list(contacts = close_schools_early),
+  time_end = time_end, increment = 1.0
+)
+
+# vaccination only (early)
+output_vacc_early <- model_default(
+  population = uk_population,
+  transmission_rate = transmission_rate,
+  infectiousness_rate = infectiousness_rate,
+  recovery_rate = recovery_rate,
+  vaccination = vacc_early,
+  time_end = time_end, increment = 1.0
+)
+
+# vaccination only (late)
+output_vacc_late <- model_default(
+  population = uk_population,
+  transmission_rate = transmission_rate,
+  infectiousness_rate = infectiousness_rate,
+  recovery_rate = recovery_rate,
+  vaccination = vacc_late,
+  time_end = time_end, increment = 1.0
+)
+
+output_npi_early_vacc_late$intervention_type <- "early npi, late vacc"
+output_npi_early$intervention_type <- "early npi"
+output_vacc_early$intervention_type <- "early vacc"
+output_vacc_late$intervention_type <- "late vacc"
+
+output <- rbind(output_baseline, output_vacc_late,
+                output_vacc_early, output_npi_early,
+                output_npi_early_vacc_late)
+
+```
+
+
+Early implementation of NPIs (`early npi`) shifts the timing of the peak of infectious individuals but does not decrease the magnitude by much. Early implementation of the vaccine (`early vacc`) is the most effective control for reducing the peak number of infectious individuals. Having an NPI in place before the vaccine is implemented (`early npi, late vacc`) is more effective than simply just implementing the vaccine late (`late vacc`).
```

Comment:
```suggestion
Early implementation of NPIs (`early npi`) delays the timing of the peak of infectious individuals but does not decrease the magnitude by much once the measure is lifted. Early implementation of the vaccine (`early vacc`) is the most effective control for reducing the peak number of infectious individuals, because it reduces the size of the susceptible group rather than just temporarily keeping infections away from it. Having an NPI in place before the vaccine is implemented (`early npi, late vacc`) is more effective than simply just implementing the vaccine late (`late vacc`).
```